### PR TITLE
Akb93 week10

### PR DESCRIPTION
**Pull request by AdamKBeck**

### CODE REVIEW

File: postr/twitter_postr.py [hunk 1]
```diff
@@ -209,13 +213,43 @@ def stream_and_graph(self, hashtags: List[str]) -> None:
 
     def graph_blob(self) -> None:
         """ Graphs a blob file for twitter sentiment """
+
+        def max_score(scores: List[float]) -> Tuple[int, float]:
```
Comment (dbgrigsby) [inline postr/twitter_postr.py]: delete this method and replace with the other one. Make the list input type and the output tuple's second type parameter a generic

File: postr/instagram_postr.py [hunk 2]
```diff
@@ -14,6 +15,9 @@
 from .instagram.instagram_key import InstagramKey
 from .api_interface import ApiInterface
 
+# Precision to truncate on a datetime object, down to the minute
+DATETIME_MINUTE_PRECISIION = 16
```
[the remark this answers is not in the document]
Comment (AdamKBeck) [reply postr/instagram_postr.py]: Good catch. Resolved.

File: postr/instagram_postr.py [hunk 3]
```diff
@@ -189,14 +193,45 @@ def _read_csv_col(colNum: int, filename: str) -> List[str]:
 
     def graph_followers(self) -> None:
         """ Graphs a blob file for twitter sentiment """
+
+        def max_followers(followers: List[int]) -> Tuple[int, int]:
+            """ Finds the max followers with its index, for global maxima plotting """
+            max_val = 0
+            max_index = 0
+            for index, val in enumerate(followers):
```
[the remark this answers is not in the document]
Comment (AdamKBeck) [reply postr/instagram_postr.py]: Good idea. Resolved.

File: postr/twitter_postr.py (hunk 1 above)
Comment (AdamKBeck) [reply postr/twitter_postr.py]: Resolved. I removed score calculations for twitter_postr.py. It made sense to find a global max for instagram_postr.py, as followers/time generally has up to a few peaks close to each other. However, for tweet polarity, scores were often in the extremes (i.e. -1 and 1), so providing an absolute max or min doesn't show much, as this happens very often.

**Comment by dbgrigsby** [comment]
also you called this week 10 but it's still week 9

**Pull request merged**